### PR TITLE
Testing vdc_dashboard chatflows

### PR DESCRIPTION
**Pull request by MElborolossy**
### Description

Describe the changes introduced by this PR and what does it affect

### Changes

- [x] Blog
- [x] Cryptpad
- [x] Discourse
- [x] Extend kubernetes
- [x] Gitea
- [x] Kubeapps
- [x] Mattermost
- [x] Monitoring
- [x] Peertube
- [x] Taiga
- [x] Website
- [x] Wiki
- [x] ZeroCI
- [x] ETCD
- [x] Digibyte

### Related Issues

#2201 

### Checklist

- [x] [Pre-commit hook is installed](https://github.com/threefoldtech/js-ng#pre-commit) to do formatting checks before committing code...etc
- [x] Tests included
- [x] Build pass
- [x] Documentation
- [x] Code format and docstrings

### CODE REVIEW

**Comment by MElborolossy** [comment]
All Tests passed, except Extend Kubernetes, because payment was down, it passed before
I will try to run it again and update the status once all passed
image

File: tests/sals/automated_chatflows/test_vdc_dashboard.py [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+        self.solution = etcd
+
+        self.info("Check that ETCD add data correctly.")
+        expiry = j.data.time.now().timestamp + 20
+        put_hello = ""
+        while j.data.time.now().timestamp < expiry:
+            rc, put_hello, err = j.sals.kubernetes.Manager._execute(
+                f"kubectl --kubeconfig {self.kube_manager.config_path} exec -it {name}-etcd-0 -- etcdctl put message Hello"
+            )
+            if not put_hello == "":
```
Comment (AhmedHanafy725) [inline tests/sals/automated_chatflows/test_vdc_dashboard.py:207]:
```suggestion
            if put_hello:
```

File: tests/sals/automated_chatflows/test_vdc_dashboard.py [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+                is_ready = True
```
Comment (AhmedHanafy725) [inline tests/sals/automated_chatflows/test_vdc_dashboard.py:208]: not used?

File: tests/sals/automated_chatflows/test_vdc_dashboard.py [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+    def test15_ExtendKubernetes(self):
+        """Test case for Extend Kubernetes cluster.
+
+        **Test Scenario**
+
+        - Deploy VDC.
+        - Get Number of Nodes before extend.
+        - Extend Kubernetes cluster.
+        - Get Number of Nodes after extend.
+        - Check that node added.
```
Comment (AhmedHanafy725) [inline tests/sals/automated_chatflows/test_vdc_dashboard.py:359]:
```suggestion
        """Test case for extending Kubernetes cluster.

        **Test Scenario**

        - Deploy VDC.
        - Get the number of nodes before extending.
        - Extend Kubernetes cluster.
        - Get the number of nodes after extending.
        - Check that node has been added.
```

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**